README.md

```diff
@@ -1,6 +1,54 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+# Installation
+
+Download the appropriate binary from the [releases](https://github.com/Brightspace/bmx/releases) page.
+
+# Commands
+
+## configure
+
+Set up the global BMX configuration file with the following command:
+```PowerShell
+bmx configure --org your-okta-organization --user your-okta-username
+```
+BMX will default to use the provided values. Okta account sessions are also saved when a configuration file is present. As such, it is not recommended to run `bmx configure` or create a configuration file on a machine with multiple users.
+
+Note: The flags provided are optional. Use `bmx configure -h` to view all available options.
+## print
+
+To setup AWS credentials in PowerShell, use:
+```PowerShell
+bmx print --account account-name --role role-name | iex
```

Comment:
The copy code button seems kind of useless when we're adding dummy flag values in

Reply:
Or it's just more work to replace